app/src/main/java/mozilla/lockbox/support/Constant.kt

```diff
@@ -11,6 +11,7 @@ import mozilla.lockbox.action.Setting
 object Constant {
     object Common {
         const val emptyString = ""
+        const val startUpLockTime: Long = 60000
```

Comment:
I would name this something like `sixtySeconds`, this name is a little unwieldy.